src/components/modals/Agenda/Agenda.tsx

```diff
@@ -118,13 +125,18 @@ export default function Agenda({
                 borderRadius: "0.25em",
                 boxShadow: 24,
                 p: 4,
-                backgroundColor: "white",
-                '[data-mui-color-scheme="dark"] &': {
-                    backgroundColor: "#181818",
-                },
+                backgroundColor: theme.palette.mode === "light" ? "white" : "#181818",
```

Comment:
Looks like we can simply replace `data-mui-color-scheme` with `data-color-scheme` and avoid adding `useTheme` everywhere 😎

Reply:
Sure, I did that now. It is a _slightly_ worse solution IMO, since it is less responsive when you change the theme. (I have to reload the page when I change my system theme) I guess that happens rarely, since I have not yet experienced it on the live version. 🤷🏻

Reply:
Hmm, mine switches instantly. Could be browser specific 🤔 But yea, not a major concern